src/cli/cli_config.rs

```diff
@@ -112,6 +113,18 @@ pub struct LockFileUpdateConfig {
     pub lock_file_usage: super::LockFileUsageConfig,
 }
 
+#[derive(Parser, Debug, Default, Clone)]
+#[clap(next_help_heading = consts::CLAP_SOLVER_OPTIONS)]
+pub struct SolverConfig {
+    /// Exclude any packages that have been created after the given date.
```

Comment:
Could you add an example on what the value should look like, this will then be represented in cli help and documentation.